Makefile

```diff
@@ -175,7 +175,7 @@ ifndef YARN
 	@echo "Could not find yarn, which is required to build the Galaxy client.\nTo install yarn, please visit \033[0;34mhttps://yarnpkg.com/en/docs/install\033[0m for instructions, and package information for all platforms.\n"
```

Comment:
Should the `ifndef YARN` check removed then?

Reply:
Not removed, but we need to reorder it I guess.  If someone's running this without a virtualenv we still need that error case, right?